docs/user_manual/site_configuration.md

```diff
@@ -190,3 +190,11 @@ easier to find that they are related because they are connected to the same ghos
 
 {% include userman-toplink.md %}
 
+{% assign sub = sub | plus: 1 %}
+{% include userman-heading.md section=page.section sub=sub title="Strict Index Checking" %}
```

Comment:
The subsection needs to be added added to the table of contents as well

Reply:
done